enn_zoo/enn_zoo/microrts/main.py

```diff
@@ -0,0 +1,95 @@
+import argparse
+from typing import Dict, Type
+from enn_zoo.microrts import GymMicrorts
+
+from entity_gym.environment import (
+    CategoricalAction,
+    SelectEntityActionMask,
+    Environment,
+    ObsSpace,
+    CategoricalActionSpace,
+    Action,
+    Observation,
+    SelectEntityAction,
+    SelectEntityActionSpace,
+)
+import numpy as np
+
+
+def print_obs(obs: Observation, total_reward: float, obs_filter: ObsSpace) -> None:
+    print(f"Reward: {obs.reward}")
+    print(f"Total reward: {total_reward}")
+    entity_index = 0
+    # for entity_type, features in obs.features.items():
+    #     for entity in range(len(features)):
+    #         print(
+    #             f"{obs.ids[entity_type][entity_index]}: {entity_type}({', '.join(map(lambda nv: nv[0] + '=' + str(nv[1]), zip(obs_filter.entities[entity_type].features, features[entity])))})"
+    #         )
+    #         entity_index += 1
+
+
+if __name__ == "__main__":
```

Comment:
We should probably just move the corresponding code in entity-gym into a function that can pass the env so you wouldn't need to copy the whole block.

Reply:
Yep, can we keep it now for debugging reasons and merge this later?